doc/api/fs.md

```diff
@@ -4594,6 +4594,21 @@ For detailed information, see the documentation of the asynchronous version of
 this API: [`fs.writev()`][].
 
 ## `fs` Promises API
+<!-- YAML
+added: v10.0.0
+changes:
+  - version: v14.0.0
+    pr-url: https://github.com/nodejs/node/pull/31553
+    description: Exposed as `require('fs/promises')`.
```

Comment:
Wondering if we should make it clearer that the API is still accessible via `require('fs').promises`. Feel free to disregard if you think it's good enough as is.